### DISCUSSION TITLE
Telegram ve discord

### DISCUSSION THREAD

**Discussion by ikincisinif** [IBAX-io/go-ibax#605] [Soru Cevap - Türkçe] [answered]
Teşegram ve discord görevini nasıl yapıyoruz

**Comment by okkoartdesign** [comment, marked as answer]
henüz bilen yok galiba

**Reply by Yusufknk** [reply]
Telegram ve discord kanalına katılmanız yeterlidir. Ödüller 1 Şubat'tan önce ayrı bir sayfa/formdan yayınlanacaktır.
